.github/unittest/linux_libs/scripts_meltingpot/environment.yml

```diff
@@ -0,0 +1,20 @@
+channels:
+  - pytorch
+  - defaults
+dependencies:
+  - pip
+  - pip:
+    - cloudpickle
+    - importlib-metadata
+    - numpy
+    - torch
+    - zipp
+    - pytest
+    - pytest-cov
+    - pytest-mock
+    - pytest-instafail
+    - pytest-rerunfailures
+    - pytest-error-for-skips
+    - expecttest
+    - pyyaml
+    - meltingpot
```

Comment:
shouldn't this be dm-meltingpot?

Reply:
oh wow you are right, there is a shadow PyPi package